h2gis-functions/src/test/java/org/h2gis/functions/io/geojson/GeojsonImportExportTest.java

```diff
@@ -568,5 +568,19 @@ public void testReadComplexFile() throws Exception {
         stat.execute("DROP TABLE IF EXISTS TABLE_POINTS_READ");
         stat.close();
     }
+
+    @Test
+    public void testReadProperties() throws Exception {
```

Comment:
You should cover all your modifications with test, for example, the method `parseObjectMetadata(JsonParser jp)` of the class GeojsonReaderDriver is not covered.